src/controllers/update/confirmation.controller.ts

```diff
@@ -28,7 +28,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       verificationCheckDays: 14,
       paymentFee: UPDATE_PAYMENT_FEE,
       isUpdate: true,
-      titleExtension: "(NOT LIVE)",
+      titleExtension: "",
```

Comment:
maybe we can remove titleExtension from the view?

Reply:
Done.